### ISSUE TITLE
Road train does nut unload on map

### ISSUE THREAD

**Issue by aawsum** [Stephan-S/FS22_AutoDrive#1132]
I have been reading a few comments on this, but did not find a solution.

I am using a road train on a 4x map, in this case the michigan farms.
When the road train goes to the unload location it does not, or not fully unload at the sell point.
My target point is set after the location so the whole road train will have passed the sell point.
I have tried the
-lode king distinction super-b
with this default hopper it only unloads the second part of the first hopper and the second part of the second hopper.

Because of this I also tried a mod
-lizard dumper road train
in this case it drives fully by the sell point and does not unload anything.

I did not find a setting for this but maybe I am overseeing something.

Can somebody advice ?

**Comment by aawsum** [comment]
Ok I messed around a bit more with the settings as I could not believe this would not work.
I changed the setting for notifying the sell point to 50 meters and changed the tipper unload option (sorry not sure what the english description is) from automatic to "center".

This seems to have done the trick as the whole truck now slows down and delivers its load.

So no bug and I provided the answer to my problem :-)
Thanks. Happy with how this is working.
